index.bs

```diff
@@ -0,0 +1,225 @@
+urlPrefix: https://www.w3.org/TR/css-box-3/; spec: CSS-BOX-3;
+    type: dfn; url: #border-box; text: border box; 
+urlPrefix: https://www.w3.org/TR/css-break-3/; spec: CSS-BREAK-3;
+    type: dfn; url: #box-fragment; text: box fragment;
+urlPrefix: https://www.w3.org/TR/cssom-view-1/; spec: CSSOM-VIEW-1;
+    type: dfn; url: #css-pixels; text: CSS pixels;
+urlPrefix: https://www.w3.org/TR/css-values-4/; spec: CSS-VALUES-4;
+    type: dfn; url: #pixel-unit; text: pixel units;
+urlPrefix: https://wicg.github.io/visual-viewport/index.html; spec: VISUAL-VIEWPORT;
+    type: dfn; url: #dom-visualviewport-width; text: visual viewport width;
+    type: dfn; url: #dom-visualviewport-height; text: visual viewport height;
+urlPrefix: https://www.w3.org/TR/CSS2/visudet.html; spec: CSS2;
+    type: dfn; url: #containing-block-details; text: initial containing block;
+    
+
+
+</pre>
+
+Introduction {#sec-intro}
+=====================
+
+<div class="non-normative">
+
+<em>This section is non-normative.</em>
+
+When the layout of a page is unstable it leads to a very bad user experience.
+This shifting of DOM elements is often due to content loading asynchronously and displacing other elements on the page.
+The layout instability metric identifies these unstable pages by measuring the layout shift fraction of each animation frame on the page, allowing the user to compute an overall instability score for the page.
+
+</div>
+
+Usage example {#example}
+------------------------
+
+<pre class="example highlight">
+    var observer = new PerformanceObserver(function(list) {
+        var perfEntries = list.getEntries();
+        for (let i = 0; i < perfEntries.length; i++) {
+            // Process entries
+            // Report back for analytics and monitoring
+            // ...
+        }
+    });
+
+    // Register observer for layout shift notifications
+    observer.observe({entryTypes: ["layoutShift"]});
+</pre>
+
+<div class="non-normative">
+
+<em>This section is non-normative.</em>
+
+<h4 dfn>End of session signal</h4>
+A "final" penalty for the user's session can be reported by listening to the <a href="https://developers.google.com/web/updates/2018/07/page-lifecycle-api#event-visibilitychange">visibilitychange event</a>, and factoring in that last value at that time.
+
+</div>
+
+Terminology {#sec-terminology}
+==============================
+An <a>Element</a>’s <dfn export>starting point</dfn> refers to the <a>element</a>'s <a>flow-relative offset</a> in the coordinate space of the <a>initial containing block</a> of the current document measured in <a>pixel units</a>.
+
+The <dfn export>visual representation</dfn> of a <a href="https://www.w3.org/TR/html401/struct/global.html#h-7.5.3">block-level</a> <a>Element</a> is its <a>border box</a>. The <a>visual representation</a> of an <a href="https://www.w3.org/TR/html401/struct/global.html#h-7.5.3">inline element</a> is the geometric union of its <a>box fragment</a>s.
```

Comment:
nit: block-level and inline treat e(E)lement casing differently, maybe just pick one?

Reply:
Done.